modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -168,7 +168,7 @@ class Form extends Component {
 				this._handleAddFieldButtonClicked.bind(this)
 			),
 			dom.on(
-				`#${namespace}ControlMenu *[title="Back"]`,
+				`#${namespace}controlMenu .sites-control-group li:nth-child(2) span.lfr-portal-tooltip`,
```

Comment:
Hey @carolmariaabb, is the property `nth-child` mandatory to address this issue? I'm worried about the addition of a new item to the menu and the Back button change its position from 2 to other, causing another rework from our side.

Reply:
Hey @natocesarrego, I agree with you. I removed the `li:nth-child(2)` property and it still works. Thanks!